models/common_data_models/recharge__common_data_models.yml

```diff
@@ -0,0 +1,81 @@
+version: 2
+
+models:
+  - name: recharge__line_item_enhanced
+    description: Add description
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - header_id
+            - line_item_index
```

Comment:
Be sure to include the `source_relation` field here. I missed this originally but you will need to include that field in this test and also in all the joins within the new model.

Reply:
Recharge is not yet a member of the union data club. 😄

Reply:
It's not cool yet haha then nevermind, disregard my comment lol